lib/argsToFindOptions.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var _Object$keys = require('babel-runtime/core-js/object/keys')['default'];
+
+Object.defineProperty(exports, '__esModule', {
+  value: true
+});
+exports['default'] = argsToFindOptions;
+
+function argsToFindOptions(args, target) {
+  var result = {},
+      targetAttributes = _Object$keys(target.rawAttributes);
+
+  if (args) {
+    _Object$keys(args).forEach(function (key) {
+      if (~targetAttributes.indexOf(key)) {
+        result.where = result.where || {};
+        result.where[key] = args[key];
+      }
+
+      if (key === 'limit' && args[key]) {
+        result.limit = args[key];
+      }
+
+      if (key === 'offset' && args[key]) {
+        result.offset = args[key];
+      }
+
+      if (key === 'order' && args[key]) {
+        if (args[key].indexOf('reverse:') === 0) {
+          result.order = [[args[key].substring(8), 'DESC']];
+        } else {
+          result.order = [[args[key], 'ASC']];
+        }
+      }
+    });
+  }
+
+  return result;
+}
+
+module.exports = exports['default'];
```

Comment:
I added the lib folder to this so i could use it straight from github for our application. I can remove it directly before its ready for merge. If you have an alternative suggestion to make this able to be installed from github so we can install versions that aren't ready for NPM for testing etc i'd love to implement that feature.

We use automated build processes on heroku so we are limited to what can be automatically installed.